### PR TITLE
New Landing Page

### PR DESCRIPTION
**Pull request by proofmancer**
### Changes:

- Preference to start fresh with the new landing page rather than modifying the existing code.
- Requirement to use TypeScript instead of the current JSX setup.
- Abstract the navigation header into a generalized component to be placed in the shared directory (or as a separate package, depending on preference). **!TODO**
- Usage of React components instead of normal SVG files.
- Prettier has been added for consistent code formatting.
- The overall architecture has been changed to reflect these updates.
- We are now using Next-SEO for SEO management.

### CODE REVIEW

File: landing/.eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {
-  "extends": "next/core-web-vitals",
-  "rules": { "react/no-unescaped-entities": 0 }
+  "extends": ["next/core-web-vitals", "next/typescript"]
```
Comment (MattNguyen) [inline landing/.eslintrc.json]: We should be sharing all config between the two projects. I recommend elevating this + prettierrc, tsconfig, etc to shared dir.

File: landing/tailwind.config.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import type { Config } from 'tailwindcss';
+
+const config: Config = {
+  content: [
```
Comment (MattNguyen) [inline landing/tailwind.config.ts:7]: What happened to the preset?

File: landing/components/ui/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Footer: React.FC = () => {
+  return (
+    <footer className="mt-4 bg-gray-800 p-4">
+      <div className="container mx-auto text-center text-gray-300">
+        &copy; {new Date().getFullYear()} MyWebsite. All rights reserved.
```
Comment (MattNguyen) [inline landing/components/ui/Footer.tsx]: Don't think this is necessary

File: landing/pages/api/hello.ts
Comment (MattNguyen) [inline landing/pages/api/hello.ts:1]: can remove the api/ directory

File: landing/pages/index.tsx
Comment (MattNguyen) [inline landing/pages/index.tsx:1]: probably should use nextjs app router instead of page

File: landing/.prettierrc
Comment (MattNguyen) [inline landing/.prettierrc:1]: We should update the linter to enforce these standards. See the list of cmds we used in our v1

https://github.com/yieldi-labs/yieldi-app/blob/main/package.json#L12-L18

File: landing/tailwind.config.ts (hunk 2 above)
Comment (proofmancer) [reply landing/tailwind.config.ts:7]: Added it back

File: landing/components/ui/Footer.tsx (hunk 3 above)
Comment (proofmancer) [reply landing/components/ui/Footer.tsx]: Placeholder till I finish the footer

File: landing/pages/index.tsx
Comment (proofmancer) [reply landing/pages/index.tsx:1]: That is what you guys prefer, or there is some other reasons?

File: landing/components/common/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// import Image from "next/image";
+import React from "react";
+
+const Hero = () => {
+  return (
+    <div className="relative px-18 bg-hero-pattern bg-cover desktop:bg-contain bg-no-repeat bg-center h-[512px] mobile-lg:h-[1024px] flex items-center justify-center">
+      {/* <Image
```
Comment (desamtralized) [inline landing/components/common/Hero.tsx]: Can we remove this commented code?

File: app/app/layout.tsx [hunk 5]
```diff
@@ -19,26 +19,41 @@ export default function RootLayout({
         <link rel="shortcut icon" href="/favicon.png" />
       </head>
       <body>
-        <div className={`fixed bottom-0 right-0 translate-x-[20%] translate-y-[20%]`}>
+        <div
+          className={`fixed bottom-0 right-0 translate-x-[20%] translate-y-[20%]`}
```
Comment (MattNguyen) [inline app/app/layout.tsx:23]: should use twmerge

File: shared/components/ui/LinkWithHoverEffect.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  column?: boolean;
+  toggleMenu?: () => void;
+}
+
+const LinkWithHoverEffect = ({
+  links,
+  column,
+  toggleMenu,
+}: LinkWithHoverEffectProps) => {
+  // const router = useRouter();
```
Comment (desamtralized) [inline shared/components/ui/LinkWithHoverEffect.tsx]: Can we remove this commented code?

File: landing/components/common/Hero.tsx (hunk 4 above)
Comment (proofmancer) [reply landing/components/common/Hero.tsx]: Removed

File: app/app/layout.tsx (hunk 5 above)
Comment (proofmancer) [reply app/app/layout.tsx:23]: We should, still didn't change that much on App, mostly focused on finishing the landing now, maybe @desamtralized can add it

File: shared/components/ui/LinkWithHoverEffect.tsx (hunk 6 above)
Comment (proofmancer) [reply shared/components/ui/LinkWithHoverEffect.tsx]: Fixed and removed

**Review by MattNguyen — APPROVED** [review]
assuming build passes

**Pull request merged**